src/Ares.php

```diff
@@ -300,6 +293,7 @@ public function findByName($name, $city = null)
 
         $records = new AresRecords();
         foreach ($elements as $element) {
+            // TODO: What is this?
             $record = new AresRecord();
             $record->setCompanyId(strval($element->ico));
             $record->setTaxId(
```

Comment:
This is because of searching records in Ares by name. I "virtually" created an instance of AresRecord because I wanted to return array of objects and not only simple array or XML. Search feature does not provide relevant information, e.g. address is not parsed.

Here is simple output of search query: http://wwwinfo.mfcr.cz/cgi-bin/ares/ares_es.cgi?obch_jm=Fridrich&obec=Ostrava&filtr=0

This is what I need to correctly create an instance with all relevant information: http://wwwinfo.mfcr.cz/cgi-bin/ares/darv_res.cgi?ICO=73263753

But if you have an idea how to make it better I'm for it. I want this library to be as useful and clear as possible.